knownprojects_build/_sql/create_corrections.sql

```diff
@@ -0,0 +1,34 @@
+-- first round of corrections -> project to project
+DROP TABLE IF EXISTS corrections_project;
+CREATE TABLE corrections_project(
+    record_id text,
+    action text,
+    record_id_match text
+);
+
+\COPY corrections_project FROM 'data/corrections/corrections_project.csv' DELIMITER ',' CSV HEADER;
+
+-- second round of corrections -> dob to project
+DROP TABLE IF EXISTS corrections_dob;
+CREATE TABLE corrections_project(
+    record_id text,
+    action text,
+    record_id_match text
```

Comment:
This should be `record_id_dob`. Same here: https://github.com/NYCPlanning/db-knownprojects-data/pull/9